modules/ml/src/main/java/org/apache/ignite/ml/util/plugin/MLPluginProvider.java

```diff
@@ -171,6 +173,14 @@ private void startModelStorage(MLPluginConfiguration cfg) {
 
         ignite.getOrCreateCache(storageCfg);
 
+        if(!ThinClientCustomQueryRegistry.registeredProcessors().contains(ModelStorateThinClientProcessor.PROCESSOR_ID)) {
```

Comment:
Maybe something like `registerIfAbsent`?

Reply:
yes, you're absolutely right!